src/FSharpPlus/Lens.fs

```diff
@@ -112,6 +112,39 @@ module Lens =
     /// Lens for the fifth element of a tuple
     let inline _5 f t = Map.InvokeOnInstance (fun x -> mapItem5 (fun _ -> x) t) (f (item5 t))
 
+    [<RequireQualifiedAccess>]
+    module List=
```

Comment:
Let's use spaces around `=`

Reply:
Agreed and resolved - I was trying to follow convention from `Map` and `Set` below 😅

Reply:
> I was trying to follow convention from Map and Set below

I know, sorry about that, it's not a good example but some contributors don't follow the conventions, from time to time I do PRs to fix it.